modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeDefinitionLocalServiceImpl.java

```diff
@@ -156,10 +156,12 @@ public ListTypeDefinition updateListTypeDefinition(
 	}
 
 	private ListTypeDefinition _addListTypeDefinition(
-			long userId, ListTypeDefinition listTypeDefinition,
-			Map<Locale, String> nameMap)
+			String externalReferenceCode, long userId,
+			ListTypeDefinition listTypeDefinition, Map<Locale, String> nameMap)
 		throws PortalException {
 
+		listTypeDefinition.setExternalReferenceCode(externalReferenceCode);
```

Comment:
Do we need to check if this ERC value is null tor duplicate to generate one instead of the one sent by the user?

Reply:
As far as I know, we don't need to check this on our side.